libs/db/badgerdb/db.go

```diff
@@ -37,6 +37,18 @@ func NewDBWithOptions(opts badger.Options) (*BadgerDB, error) {
 	return &BadgerDB{db: db}, nil
 }
 
+// NewInMemoryDB creates a light weight in-memory BadgerDB.
+// Mainly useful for unit-tests.
+func NewInMemoryDB() (*BadgerDB, error) {
+	opts := badger.DefaultOptions("")
+	opts.InMemory = true
```

Comment:
@marbar3778 you guys should do the same for unit tests over at tendermint (if this isn't already planned or even done anyways).

Reply:
Will try that one too. It's probably still more lightweight.